PowerShell Scanners/Hosts File/Hosts File.ps1

```diff
@@ -0,0 +1,30 @@
+#Get the contents of the hosts file
+$HostFile = Get-Content -Path "C:\Windows\System32\drivers\etc\hosts"
+
+#Add valid host entries into an arraylist
+#Collect the IP address, hostname, and comments
+[System.Collections.ArrayList]$HostsFileEntries = @()

```

Comment:
Small optimization: you don't need to collect the PSCustomObjects into an array. You can just emit each one as it's built.

Reply:
I committed the change below